scripts/getLatestDeadlineVersion.ts

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+const mod = require('../packages/aws-rfdk/lib/core/lambdas/nodejs/lib/version-provider');
```

Comment:
was there something that prevented us from being able to use an import instead?

```ts
import { VersionProvider } from '../packages/aws-rfdk/lib/core/lambdas/nodejs/lib/version-provider';

Reply:
Yeah. I didn't think it's worth digging into, but I was getting node errors when trying that syntax.